### PR TITLE
browsers: add Flatpak support for popular browsers

### PR DESCRIPTION
**Pull request by apatel762**
This commit adds support for scanning Flatpak installations of:

- Brave Browser
- Google Chrome
- Chromium
- Firefox

The README has also been updated to add some instructions to help first-time developers get started with the project.

### CODE REVIEW

File: browserexport/browsers/common.py [hunk 1]
```diff
@@ -117,7 +119,8 @@ def handle_glob(bases: Sequence[Path], stem: str, recursive: bool = False) -> Pa
     recur_desc = "recursive" if recursive else "non recursive"
     logger.debug(f"Glob {bases} with {stem} ({recur_desc}) matched {dbs}")
     if len(dbs) > 1:
-        raise RuntimeError(errmsg.format(dbs))
+        human_readable_db_paths: str = "\n".join([str(db) for db in dbs])
+        raise RuntimeError(errmsg.format(human_readable_db_paths))
```
Comment (apatel762) [inline browserexport/browsers/common.py:123]: The error message looks something like this now:

```
RuntimeError: Expected to match a single database, but found:
/var/home/apatel/.var/app/com.brave.Browser/config/BraveSoftware/Brave-Browser/Default/History
/var/home/apatel/.var/app/com.brave.Browser/config/BraveSoftware/Brave-Browser/Guest Profile/History
/var/home/apatel/.var/app/com.brave.Browser/config/BraveSoftware/Brave-Browser/System Profile/History

You can use the --profile argument to select one of the profiles/match a particular file
```
(the difference being that the paths are printed on new lines, which makes them a little easier to read)

Comment (seanbreckenridge) [reply browserexport/browsers/common.py:123]: yep looks good, thanks

**Comment by seanbreckenridge** [comment]
Will probably add a note to make the `.venv` optional, as I don't believe there are any especially specific dependencies here which would require one (if there are, let me know), but its good to have an example anyways :+1:

**Comment by seanbreckenridge** [comment]
Updated the docs abit and pushed version `2.10` to pypi.

Thanks again :+1:

**Comment by apatel762** [comment]
> Updated the docs a bit and pushed version `2.10` to pypi.
> 
> Thanks again +1

Updated docs look good to me :slightly_smiling_face: Thanks

**Pull request merged**